app/common/directives/category-selector.html

```diff
@@ -12,13 +12,22 @@
     <div class="form-field checkbox" ng-repeat="(index, category) in categories" ng-class="{ overflow : ($index > 1) }" ng-if="category.parent_id === null">
         <label>
             <input
+                ng-show="category.children.length === 0"
                 type="checkbox"
                 checklist-value="category.id"
                 ng-disabled="disabledCategories[category.id]"
                 value="category.id"
                 checklist-model="selected"
                 checklist-change="changeCategories()"
             >
+            <input
+                    ng-show="category.children.length > 0"
+                    type="checkbox"
+                    checklist-value="category.id"
+                    ng-disabled="true"
+                    value="category.id"
+                    checklist-model="selectedParents"
+            >
```

Comment:
It looks like the categories doesn't have access to the whole child-objects when I am in the survey-field-category-selector. Not sure why that happened. It doesn't happen in develop :/

Reply:
fixed

Reply:
> I also didnt exactly understand step nb 6 in the test-checklist: Select only the newly created child category as a filter so maybe I have missed testing something?
this is in another ticket now, we don't have a way to add children in add post screen